scripts/src/components/organisation-form.js

```diff
@@ -106,13 +105,59 @@ var CreateOrg = React.createClass({
     }
   },
 
-
-  _handleReferenceLinksChange(newData, errors) {
-    if (errors.length == 0) {
-      this.setState({referenceLinks: newData, submitDisabled: false});
-    } else {
-      this.setState({referenceLinks: newData, submitDisabled: true});
+  /**
+   * Parses each reference link and checks for url
+   * placeholders that are expected
+   *
+   * @param {object} links
+   */
+  _parseReferenceLinks(links) {
+    const placeholders = ['{source_id}'],
+          errors = {'reference_links': []};
+    for (let idType in links) {
+      if (idType && links[idType]) {
+        placeholders.forEach(ph => {
+          if (links[idType].indexOf(ph) === -1) {
+            const errorMsg = `The URL for asset ID type ${idType} is missing the placeholder ${ph}`;
```

Comment:
Should this be source id type? Since asset corresponds to the internal id that we assign to things not the source id that companies define